### PR TITLE
[docs](hive)add hadoop.username config item

### PR DESCRIPTION
**Pull request by chenlinzhong**
# Proposed changes

Issue Number: close #xxx

## Problem summary

add `hadoop.username`  property  ,which is used to visit hdfs when your authentication type is sample

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

**Review by adonis0147 — DISMISSED** [review]
LGTM

File: docs/en/docs/ecosystem/external-table/hive-of-doris.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ PROPERTIES (
     - `hive.metastore.uris`: Hive Metastore service address
     - `database`: the name of the database to which Hive is mounted
     - `table`: the name of the table to which Hive is mounted
+    - `hadoop.username`: the username to visit hdfs, when the authentication type is sample need to set
```
Comment (dataalive) [inline docs/en/docs/ecosystem/external-table/hive-of-doris.md:151]: sample -> simple

File: docs/zh-CN/docs/ecosystem/external-table/hive-of-doris.md [hunk 2]
```diff
@@ -148,6 +148,7 @@ PROPERTIES (
     - `hive.metastore.uris`：Hive Metastore 服务地址
     - `database`：挂载 Hive 对应的数据库名
     - `table`：挂载 Hive 对应的表名
+    - `hadoop.username`: 访问hdfs用户名,当认证为sample时需要
```
Comment (dataalive) [inline docs/zh-CN/docs/ecosystem/external-table/hive-of-doris.md:151]: sample -> simple

Comment (chenlinzhong) [reply docs/zh-CN/docs/ecosystem/external-table/hive-of-doris.md]: fix

File: docs/en/docs/ecosystem/external-table/hive-of-doris.md (hunk 1 above)
Comment (chenlinzhong) [reply docs/en/docs/ecosystem/external-table/hive-of-doris.md]: fix

File: docs/en/docs/ecosystem/external-table/hive-of-doris.md [hunk 3]
```diff
@@ -148,7 +148,7 @@ PROPERTIES (
     - `hive.metastore.uris`: Hive Metastore service address
     - `database`: the name of the database to which Hive is mounted
     - `table`: the name of the table to which Hive is mounted
-    - `hadoop.username`: the username to visit hdfs, when the authentication type is sample need to set 
+    - `hadoop.username`: the username to visit hdfs, when the authentication type is simple need to set
```
Comment (adonis0147) [inline docs/en/docs/ecosystem/external-table/hive-of-doris.md:151]:
```suggestion
    - `hadoop.username`: the username to visit HDFS (need to specify it when the authentication type is simple)
```

**Review by adonis0147 — CHANGES_REQUESTED** [review]

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**